deploy/operator/libvirt_disks.sh

```diff
@@ -1,6 +1,10 @@
 __dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 source ${__dir}/utils.sh
 
+function print_help() {
+  echo "Usage: DISKS=\$(echo sd{b..f}) bash ${0} (create|destroy|print_help)"
```

Comment:
sorry about that
couldn't find a way to generate it without filtering functions from imports
and mentioning it explicitly seems better than mentioning other unrelated imports implicitly